packages/components/src/circular-option-picker/types.ts

```diff
@@ -41,21 +41,40 @@ export type CircularOptionPickerProps = {
 	 */
 	children?: ReactNode;
 	/**
-	 * Whether the keyboard interaction should wrap around.
+	 * Whether the control should present as a set of buttons,
+	 * each with its own tab stop.
 	 *
-	 * @default true
+	 * @default false
 	 */
-	loop?: boolean;
-} & (
-	| {
-			'aria-label': string;
-			'aria-labelledby'?: never;
-	  }
-	| {
-			'aria-label'?: never;
-			'aria-labelledby': string;
-	  }
- );
+	asButtons?: boolean;
+};
+
+export type ListboxCircularOptionPickerProps =
+	CommonCircularOptionPickerProps & {
+		asButtons: false | undefined;
+		/**
+		 * Prevents keyboard interaction from wrapping around.
+		 * Only used when `asButtons` is not true.
+		 *
+		 * @default false
+		 */
+		disableLooping?: boolean;
```

Comment:
It's still [listed under `Unreleased`](https://github.com/WordPress/gutenberg/blob/b05bee7e2e8eefb2be06c228002e4f9783847747/packages/components/CHANGELOG.md#unreleased), so I'm assuming that means there's no public consumption yet.

Either way, [it can't be used outside of `@wordpress/components`](https://github.com/WordPress/gutenberg/blob/b05bee7e2e8eefb2be06c228002e4f9783847747/packages/components/src/circular-option-picker/README.md?plain=1#L3-L5), so any breaking changes should be constrained to the package, and we'd be aware of them.

Reply:
Sounds good. Forgot to ask before — what is the reason for the prop name change? I found `loop` simpler, the double negation of `disableLooping="false"` may require a moment to understand

Reply:
I guess it's just more my preference for value-less boolean props, as with HTML.

`<Component disableLooping>` feels cleaner/more natural than `<Component loop={false}>`, when the default value is `true`. But I can go either way – happy to change it back.